tests/e2e/start-fullscreen.test.js

```diff
@@ -0,0 +1,10 @@
+import { ClientFunction, Selector } from 'testcafe';
+import BookReader from './models/BookReader';
+
+const { BASE_URL } = process.env;
+
+fixture `Start fullscreen`.page `${BASE_URL}immersion-mode.html`;
+
+test('starts in full-screen when `startFullScreen: true` set', async t => {
+  await t.expect(Selector('#BookReader').getAttribute('class')).match(/br-ui-full/);
```

Comment:
This should be do-able as a unit test; can we do it there? E2E tests are kind of expensive.